ccpm/scripts/pm/epic-merge/execute-merge.sh

```
@@ -0,0 +1,56 @@
+git pull origin main
+
+# Generate feature list
+feature_list=""
+if [ -d ".claude/epics/$ARGUMENTS" ]; then
+  cd ".claude/epics/$ARGUMENTS" || exit 1
+  for task_file in [0-9]*.md; do
+    [ -f "$task_file" ] || continue
+    task_name=$(grep '^name:' "$task_file" | cut -d: -f2 | sed 's/^ *//')
+    feature_list="$feature_list\n- $task_name"
+  done
+  cd - > /dev/null
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)